docs/pywbemcli/cmdlineinterface.rst

```diff
@@ -225,6 +226,25 @@ user):
 
     pywbemcli> !EXTERNAL-COMMAND
 
+The general options may be included on the interactive command line to
+modify the general options entered in the initial command line for pywbemcli.
```

Comment:
The use of "modify" may be interpreted such that the changes persist. That is not the case however, I suggest the use of "override" instead.

Reply:
Good catch DONE